packages/@uppy/tus/src/index.js

```diff
@@ -327,7 +331,11 @@ module.exports = class Tus extends Plugin {
       })
 
       socket.on('success', (data) => {
-        this.uppy.emit('upload-success', file, data, data.url)
+        const uploadResp = {
+          uploadURL: data.url
+        }
+
+        this.uppy.emit('upload-success', file, uploadResp)
         this.resetUploaderReferences(file.id)
```

Comment:
Should `data` be passed as `body` here? 🤔

Reply:
I think no since tus servers don't have useful information in a response body anyway